openday_scavenger/puzzles/shuffleanagram/templates/index.html

```diff
@@ -2,6 +2,9 @@
 <html lang="en">
 
 <head>
+    <title>Anagram Puzzle</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
@luigiferiani this was the only puzzle that didn't have a viewport tag. I'm hoping it doesn't mess up the layout, I'm trying to work out how to test on my phone on my home network at the moment.

Reply:
🤦 Oops, I just realised you fixed that in the other PR. Nothing to see here!